### PR TITLE
Dcat serializer

### PR DESCRIPTION
**Pull request by mesemus**

### CODE REVIEW

File: common/datasets/serializers/dcat/schema.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    class Meta:
+        unknown = ma.RAISE
+    
+    schemaVersion = ma_fields.String()
+    identifiers = ma_fields.List(ma_fields.Nested(IdentifierSchema))
+    publisher = ma_fields.String()
+    types = ma_fields.Nested(lambda: ResourceTypeSchema)
+
+    def __init__(self, *args, **kwargs):
+        url_value = kwargs.pop('url', None)
```
Comment (mesemus) [inline common/datasets/serializers/dcat/schema.py]: suspicious - url_value - why is it here?

File: common/datasets/serializers/dcat/schema.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        super().__init__(*args, **kwargs)
+        if url_value:
+            if 'identifiers' not in self.fields:
+                self.fields['identifiers'] = ma_fields.List(ma_fields.Nested(IdentifierSchema))
+            self.fields['identifiers'].default = [IdentifierSchema.create_identifier(url_value)]
+        if 'version' in kwargs:
+            del kwargs['version']
+
+    def transform_metadata(self, metadata):
```
Comment (mesemus) [inline common/datasets/serializers/dcat/schema.py]: use @pre_load from marshmallow to execute code before the .load is called, no need to call this method explicitly

File: common/datasets/serializers/dcat/schema.py (hunk 1 above)
Comment (pragerdom) [reply common/datasets/serializers/dcat/schema.py]: Residual constructor from refactoring (moved to transform_metadata) - will be removed

File: common/datasets/serializers/dcat/schema.py (hunk 2 above)
Comment (pragerdom) [reply common/datasets/serializers/dcat/schema.py]: Added @pre_load to execute the modifications

**Pull request merged**